docs/snippets/all/views/spatial3d.py

```diff
@@ -22,6 +22,15 @@
         name="3D Scene",
         # Set the background color to light blue.
         background=[100, 149, 237],
+        # Configure the line grid.
+        line_grid=rrb.archetypes.LineGrid3D(
+            visible=True,  # The grid is enabled by default, but you can hide it with this property.
+            spacing=0.1,  # Makes the grid more fine-grained.
+            # By default, the plane is inferred from view coordinates setup, but you can set arbitrary planes.
+            plane=rr.components.Plane3D((0, 0, 1), -5.0),
```

Comment:
Tempted to switch  to `rr.components.Plane3D.XY` here for visibility. Astute users will infer that `Plane3D` might have a constructor for more custom things.

Reply:
I'll add it to a comment - this scene is just so well suited for a a -5 offset :/

Reply:
or maybe add an offset method to it to change it!

Reply:
yes, I thought the same!